### PR TITLE
sioniks - dom_done 

### PR DESCRIPTION
**Pull request by sioniks**
### DOM

[Demo](https://sioniks.github.io/kottans_frontend/task_6/js-dom/) |
[Code base](https://github.com/sioniks/kottans_frontend/tree/master/task_6/js-dom)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/sioniks/js-dom/main.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+document.addEventListener('DOMContentLoaded', function () {
+  let pageContent = document.querySelector('.page-content');
+  pageContent.innerHTML = contentData.ukr;
+
+  let leftMenu = document.querySelector('.left-menu');
+  leftMenu.addEventListener('click', function () {
+    this.classList.contains('is-active') ? this.classList.remove('is-active') : this.classList.add('is-active');
```
Comment (zonzujiro) [inline submissions/sioniks/js-dom/main.js]: `classList.toggle`

File: submissions/sioniks/js-dom/main.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+document.querySelector('.menu').addEventListener('click', function (evt) {
+  let pageContent = document.querySelector('.page-content');
+  evt.preventDefault();
+  if (evt.target.nodeName.toLowerCase() === 'a') {
+    pageContent.classList.add('hide');
+    let thisTheme = evt.target.dataset.theme;
+    setTimeout(() => {
+      pageContent.innerHTML = contentData[thisTheme];
+      pageContent.classList.remove('hide');
+    }, 500)
```
Comment (zonzujiro) [inline submissions/sioniks/js-dom/main.js]: Magic numbers are [bad](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad)

File: submissions/sioniks/js-dom/main.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  });
+  let page = document.querySelector('.page-content');
+  page.addEventListener('click', function () {
+    leftMenu.classList.contains('is-active') ? leftMenu.classList.remove('is-active') : null;
+  });
+});
+
+document.querySelector('.menu').addEventListener('click', function (evt) {
+  let pageContent = document.querySelector('.page-content');
+  evt.preventDefault();
```
Comment (zonzujiro) [inline submissions/sioniks/js-dom/main.js]: Why do you need `preventDefault` here?

File: submissions/sioniks/js-dom/style.css [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  left: 0;
+}
+
+/* .left-menu:hover {
+  left: 0;
+}
+
+.left-menu:hover .menu-icon {
+  display: none;
+} */
```
Comment (zonzujiro) [inline submissions/sioniks/js-dom/style.css]: Usually, you should avoid commented parts in your pull requests.

File: submissions/sioniks/js-dom/main.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  token: '<article> <div class = "container" ><h1 class = "article-header" > Жетоны </h1> <p class = "text" >Жетоны часто выполняют роль альтернативных средств оплаты и применяются для оплаты таких услуг как: мойки, метро, автоматы продажи воды, туалеты, почта, телефонные автоматы и прочее </p> <div class = "img-wrap" ></div><div class = "img-block" ><h3 class = "img-header" > спец жетон министерства торговли </h3><img src = "img/token2.jpg" alt = "спец жетон министерства торговли" ></div> <div class = "img-block" ><h3 class = "img-header" > спец жетон министерства торговли </h3> <img src = "img/token3.jpeg" alt = "спец жетон министерства торговли" ></div> <div class = "img-block" ><h3 class = "img-header" > жетон для оплаты услуг ресторана </h3> <img src = "img/token4.jpg" alt = "жетон для оплаты услуг ресторана" ></div> <div class = "img-block" ><h3 class = "img-header" > жетон метрополитена </h3> <img src = "img/token5.jpg" alt = "жетон метрополитена" ></div></div> </div> </article>'
+}
```
Comment (zonzujiro) [inline submissions/sioniks/js-dom/main.js]: You can make this part more readable with string templates. Then you will get something like this:

```html
<article>
  <div class="container">
    <h1 class="article-header">Жетоны</h1>
    ...
  </div>
</article>
```

File: submissions/sioniks/js-dom/main.js [hunk 6]
```diff
@@ -0,0 +1,34 @@
+document.addEventListener('DOMContentLoaded', function () {
```
Comment (zonzujiro) [inline submissions/sioniks/js-dom/main.js]: Why not `window.onload`?

File: submissions/sioniks/js-dom/main.js (hunk 1 above)
Comment (sioniks) [reply submissions/sioniks/js-dom/main.js]: done

File: submissions/sioniks/js-dom/main.js (hunk 2 above)
Comment (sioniks) [reply submissions/sioniks/js-dom/main.js]: i fixed it

File: submissions/sioniks/js-dom/main.js (hunk 3 above)
Comment (sioniks) [reply submissions/sioniks/js-dom/main.js]: this prevents page reloading by clicking on the link

File: submissions/sioniks/js-dom/main.js (hunk 5 above)
Comment (sioniks) [reply submissions/sioniks/js-dom/main.js]: thanks, I forgot about it

File: submissions/sioniks/js-dom/main.js (hunk 6 above)
Comment (sioniks) [reply submissions/sioniks/js-dom/main.js]: I thought it would be the best solution.

Comment (zonzujiro) [reply submissions/sioniks/js-dom/main.js]: But why? 😄

**Comment by zonzujiro** [comment]
@sioniks pls, resolve outdated discussions

File: submissions/sioniks/js-dom/main.js (hunk 1 above)
Comment (zonzujiro) [reply submissions/sioniks/js-dom/main.js]: I don't see changes

**Pull request merged**